gocourse03/drugsTask/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"github.com/brianvoe/gofakeit/v7"
+	"time"
+)
+
+/*
+Треба відсортувати ящик ліків за датою виробництва.
+Застарілі
+Нові
+яким півроку
+*/
+
+type Drugs struct {
+	Name             string
+	ManufacturerDate time.Time
```

Comment:
Зазвичай ось так поля називають:
```suggestion
	ManufacturedAt time.Time
```

https://google.github.io/styleguide/go/decisions#repetition